app/crud/prediction.py

```
@@ -0,0 +1,45 @@
+from datetime import datetime, timedelta, UTC
```

Ruff (0.12.2)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)